src/components/Button/Button.tsx

```diff
@@ -0,0 +1,43 @@
+import { FC } from "react";
+
+import cn from "classnames";
+
+import cls from "./Button.module.scss";
+import { Loader, LoaderSize } from "../Loader";
+
+export type ButtonProps = React.PropsWithChildren<{
+  loading?: boolean;
+}> &
+  React.ButtonHTMLAttributes<HTMLButtonElement>;
+
+export const Button: FC<ButtonProps> = ({
+  className,
+  children,
+  loading,
+  disabled,
+  ...otherProps
+}) => (
+  <>
+    <button
+      className={cn(
+        className,
+        cls.button,
+        loading && cls.loading,
+        disabled && cls.button_disabled
+      )}
+      disabled={loading ? true : disabled}
```

Comment:
думаю, можно сделать так, покороче:
```suggestion
      disabled={loading || disabled}
```